src/views/Home/components/ChapelProgress/index.js

```diff
@@ -35,25 +36,28 @@ export default class ChapelPsrogress extends Component {
         backgroundColor: [gordonColors.primary.blue],
       }],
       labels: [
-        'Chapel Events Attended',
-        'Events Remaining',
+        'CL&W Credits',
+        'CL&W Credits Remaining',
       ],
     };
     if (this.state.loading === true) {
       content = <GordonLoader />;
     } else {
       content = <Doughnut data={data} />;
+      if (current === 1) {
```

Comment:
It is not grammatically correct to pluralize "CL&W" (it would be "Christian Life and Worships").

Can we change the subheader from `${current} CL&W` and `${current} CL&W's` to `${current} CL&W Credit` and `${current} CL&W Credits`?